bundler/lib/bundler/templates/newgem/.github/workflows/main.yml.tt

```diff
@@ -0,0 +1,18 @@
+name: Ruby
+
+on: [push,pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: <%= ::Gem::Version.new(RUBY_VERSION).segments[0..1].join(".") %>.x
```

Comment:
The new action supports full ruby version locking, so you can say `RUBY_VERSION` here like in the other templates.